rector.php

```diff
@@ -2,7 +2,6 @@
 
 declare(strict_types=1);
 
-use Rector\CodeQuality\Rector\ClassMethod\DateTimeToDateTimeInterfaceRector;
```

Comment:
can we have cs-fixer configured to run over this file?

Reply:
(and others in root directory?)

Reply:
((and also phpstan))

Reply:
https://github.com/Lendable/composer-license-checker/pull/63/commits/9e1d6e9d5122dd6b871335b48992ce010c3e36a5